tests/resources/test_material_run.py

```diff
@@ -36,6 +37,20 @@ def test_register_material_run(collection, session):
     assert "<Material run 'Test MR 123'>" == str(registered)
 
 
+def test_nomutate_taurus(collection, session):
+    """When registering a Taurus object, the object should not change"""
+    # Given
+    session.set_response(MaterialRunDataFactory(name='Test MR mutation'))
+    before = TaurusRun(name='Main')
```

Comment:
Give the material run a uid and the test should pass

Reply:
Noted and implemented.